webserver/views/api/v1/core.py

```diff
@@ -182,10 +186,15 @@ def get_data_for_multiple_recordings(collect_data):
     recordings = check_bad_request_for_multiple_recordings()
 
     recording_details = {}
-
+    map_keys = request.args.get("map")
```

Comment:
One thing that we try and do in our code is separate input/out for the user with the calculation or retrieval of data.
This makes it easier to test. This means that you should get the value of the `map` parameter in 
 the `get_many_higlevel` view, and pass it to this method as an argument. This will also fix the bug of passing the parameter to the get_many_lowlevel view